tests/unit/nr-hooks.tap.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const tap = require('tap')
+const utils = require('@newrelic/test-utilities')
+const hooks = require('../../nr-hooks')
+
+utils.tap
+
+tap.test('Testing koa instrumentation via hooks', function (t) {
```

Comment:
this is fine but I was suggesting to use the nr-hooks file to replace the registerInstrumentation in the versioned tests for example [here}(https://github.com/newrelic/node-newrelic-koa/blob/main/tests/versioned/koa.tap.js#L24).